src/test/java/net/minecraftforge/debug/DataGeneratorTest.java

```diff
@@ -126,11 +129,11 @@ public ItemModels(DataGenerator generator)
         protected void registerBuilders()
         {
             getBuilder("test_generated_model")
-                    .parent(new ResourceLocation("item/generated"))
+                    .parent(new UncheckedModelFile("item/generated"))
                     .texture("layer0", "block/stone");
             
             getBuilder("test_block_model")
-                    .parent(new ResourceLocation("block/block"))
+                    .parent(new ExistingModelFile("block/block"))
```

Comment:
Add the blockstate test here too.

Reply:
I'll wait until https://github.com/tterrag1098/MinecraftForge/pull/5#discussion_r332282765 is resolved and then copy the example from `ForgeMod`, I don't want to have to rewrite the same generator twice.